.github/scripts/trymerge.py

```diff
@@ -868,10 +869,25 @@ def merge_on_green(pr_num: int, repo: GitRepo, dry_run: bool = False, timeout_mi
         current_time = time.time()
         elapsed_time = current_time - start_time
 
-
         pr = GitHubPR(org, project, pr_num)
         try:
-            return pr.merge_into(repo, dry_run=dry_run)
+            if all_green:
+                checks = pr.get_checkrun_conclusions()
```

Comment:
I wonder how long this will actually run before it runs into an API limit?

Reply:
It should only check it every minute so max, we're using 60/hour per ongoing merge so maybe an additional 300-500 requests per hour out of 5k or 15k? I think it just uses the GITHUB_TOKEN so yea it would affect rate limits globally. 

We can potentially set it to 5 minutes and not affect merge latency that much to reduce the rate limiting.

Reply:
So would this bypass the reviewer check among other checks?